components/footer.js

```diff
@@ -0,0 +1,109 @@
+export default function Footer() {
+  let year = new Date().getFullYear();
+
+  return (
+    <footer className="footer bg-white pt-1 text-gray">
+      <div className="container mx-auto px-6 border-t-2">
+        <div className="text-center py-6">
+          <div className="inline-flex text-black">
+            <a href="" className="px-4">
+              <svg
```

Comment:
can you extract all these svg icons into svg files inside public? then you can link them. It makes your file more readable